### ISSUE TITLE
Manual Partitioning install is formating all the partitions of the disk

### ISSUE THREAD

**Issue by Faugus** [archlinux/archinstall#2055]
After #1971 was marked as closed, I tried to install Arch again and now the installer is formating all the other partitions of the disks aswell.

I just marked to wipe the 2 partitions for the installation (/boot and /) and the installer also formated my Windows and data partitions.

![PXL_20230914_144835253](https://github.com/archlinux/archinstall/assets/112667550/1b44e6db-6656-4fc6-a783-c8dc79b990d5)

**Comment by Torxed** [comment]
Did you try with the GitHub version or what's default on the Arch ISO?

**Comment by Faugus** [comment]
@Torxed I tried with the default and updating it with `pacman -Sy archinstall`.

**Comment by Torxed** [comment]
Ah, that won't do anything until we do a release. The PR fixed the issue so the issues get closed to filter out tasks that are done.

**Comment by svartkanin** [comment]
@Faugus would you mind trying the latest master if you can? 
I do think as well that the issue is resolved but this is a very big concern and should definitely not happen so checking this again would be good:)
I'll do some testing on my end as well

**Comment by Faugus** [comment]
@svartkanin what do I do?
![PXL_20230915_194634887](https://github.com/archlinux/archinstall/assets/112667550/4f9db600-4044-4012-8df8-401813e07518)

**Comment by svartkanin** [comment]
Just install git otherwise there isn't enough space `paan -Sy git` that's all you should need.
Then these instructions to get the latest master https://github.com/archlinux/archinstall#using-a-live-iso-image

**Comment by Faugus** [comment]
@svartkanin Can't install it. It needs the other packages.

**Comment by svartkanin** [comment]
Are you on the latest iso?
Could you show the error you get?

**Comment by Faugus** [comment]
@svartkanin Yes, I'm on the latest ISO.

Without the `pkgconf`
![PXL_20230915_203942599](https://github.com/archlinux/archinstall/assets/112667550/bc6533bd-8c85-44a0-9186-a5dcde417c33)

With the `pkgconf` installed
![PXL_20230915_204013900](https://github.com/archlinux/archinstall/assets/112667550/b44b44c9-af97-4268-b8c0-c47cc2ce6655)

**Comment by svartkanin** [comment]
Ahh right, my bad you don't need the last step in the instructions. After cloning the repository and cd into it you can simply run it with `python -m archinstall`

**Comment by Faugus** [comment]
@svartkanin it went fine now. It didn't rekt my other partitions. Thank you.
![PXL_20230915_211257681](https://github.com/archlinux/archinstall/assets/112667550/0f1e9cdc-64b5-46cf-a86b-23127991674f)

**Comment by svartkanin** [comment]
Fantastic thanks for confirming!!

**Comment by svartkanin** [comment]
@Torxed shall we release a new version with the changes as this was a quite critical bug

**Comment by Faugus** [comment]
@svartkanin one last thing: I don't have a swap partition anymore and that was causing the #1971 problem. So I can't confirm if it is fixed.

**Comment by Torxed** [comment]
> @Torxed shall we release a new version with the changes as this was a quite critical bug

I think we should!
Also you can increase the ramdisk size to get around the space issue with pacman.

You simply add a kernel boot parameter
